.gitattributes

```diff
@@ -1 +1,2 @@
 *.bin filter=lfs diff=lfs merge=lfs -text
+proto/src/gen/proto_descriptor.bin filter=lfs diff=lfs merge=lfs -text
```

Comment:
Nit: no changes required to `.gitattributes`; the preexisting `*.bin` rule is sufficient for our needs:

```
❯ fd -e bin
proof-params/src/gen/output_pk.bin
proof-params/src/gen/spend_pk.bin
proof-params/src/gen/swap_pk.bin
proto/src/gen/proto_descriptor.bin
```